plugins/inputs/kafka_consumer/sample.conf

```diff
@@ -6,6 +6,14 @@
   ## Topics to consume.
   topics = ["telegraf"]
 
+  ## Topic regular expressions to consume.  Matches will be added to topics.
+  # topic_regexps = [ "*test", "metric[0-9A-z]*" ]
+
+  ## Topic discovery refresh interval.  If topic_regexps is set, then
+  ## every interval, topics will be rescanned to see if any new matches
+  ## are found.
+  # topic_refresh_interval = "60s"
```

Comment:
Commented out config options should express the default value, this seems not the case.

Reply:
Fixed by @srebhan below.